pkg/statestore/leveldb/leveldb_test.go

```diff
@@ -48,3 +48,33 @@ func TestPersistentStateStore(t *testing.T) {
 		return store
 	})
 }
+
+func TestGetSchemaName(t *testing.T) {
+	dir, err := ioutil.TempDir("", "statestore_test")
```

Comment:
I would suggest to use t.TempDir for convenience. Not just here but in all new code in this PR (6 instances).